src/resources/user_address/model/index.model.js

```diff
@@ -1,7 +1,26 @@
 import UserAddress from './user_address.model';
 
-async function createUserAddress(address) {
+export async function createUserAddress(address) {
   return UserAddress.query().insert(address).returning('*');
 }
 
-export default createUserAddress;
+export async function patchUserAddress(id, address) {
+  return UserAddress.query()
+    .where({ id })
+    .patch({ ...address, updated_at: UserAddress.fn.now() })
+    .returning('*')
+    .first();
+}
+export async function fetchUserAddresses(userId) {
+  return UserAddress.query().select('*').where({ user_id: userId });
+}
+export async function fetchOneUserAddress(userId, addressId) {
+  return UserAddress.query()
+    .select('*')
+    .where({ user_id: userId, id: addressId })
+    .first();
```

Comment:
Will this query return the latest address or it just returns a random address of the user?

Reply:
returns the first one it finds